index.html

```diff
@@ -0,0 +1,606 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="./images/tnw-logo.png" type="image/x-icon">
+  <title>TNW</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+  <link href="https://fonts.googleapis.com/css?family=Rajdhani&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <!-- ====== Header Section Starts ====== -->
```

Comment:
- [ ] I think your code is so well structured that you don't need these comments to read it. You can leave a line between each section to make it easier to find, but you are using semantic HTML so well, that you just need to read the tag and you'll find what you need.